en/includes/guides/organization-management/email-domain-based-organization-discovery.md

```diff
@@ -1,193 +1,212 @@
+    To do so,
 
-        {% if (product_name == "WSO2 Identity Server") %}
+    1. Login to the root organization Console.
+    2. Navigate to **Login & Registration**, and under **Organization Settings**, click **Organization Discovery**.
+    3. Turn the toggle on to enable email domain based organization discovery.
 
-        - Federated authentication and Just-In-Time (JIT) provisioning for child organizations are restricted to email domains mapped to those child organizations.
+        ![Enable email domain based organization discovery]({{base_path}}/assets/img/guides/organization/manage-organizations/enable-email-domain-based-organization-discovery.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
 
-        {% endif %}
-{% if (product_name == "WSO2 Identity Server") %}
+    {% if (product_name == "WSO2 Identity Server") %}
 
-4. Select the **Email domain discovery for self-registration** checkbox if you want to allow users to discover and self-register in child organizations based on their email domain.
+    4. Select the **Email domain discovery for self-registration** checkbox if you want to allow users to discover child organizations based on their email domain and self-register in them.
 
-    !!! note
-        To use this capability, self-registration must be enabled in the child organizations. Currently, enabling self-registration for child organizations via the console is not supported. Instead, you need to add the following configuration to the `deployment.toml` file located in the `<IS_HOME>/repository/conf` directory to enable self-registration server-wide.
+        !!! note
+            For this to work, child organizations should have self-registration enabled. Currently, you can enable self-registration for all child organizations by adding the following configuration to the `<IS_HOME>/repository/conf/deployment.toml` file.
 
-        ```
-        [identity_mgt.user_self_registration]
-        allow_self_registration = true
-        ```
+            ```
+            [identity_mgt.user_self_registration]
+            allow_self_registration = true
+            ```
 
-{% endif %}
+    {% endif %}
+
+    5. [Map email domains to organizations](#map-email-domains-to-organizations).
 
-    ![Enable email domain based organization discovery]({{base_path}}/assets/img/guides/organization/manage-organizations/enable-email-domain-based-organization-discovery.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+=== "Using the API"
 
-### Using the API
+    !!! note "Before you begin"
 
-1. [Get an access token]({{base_path}}/apis/#oauth-based-authentication) with the `internal_organization_config_add` scope.
+        Ensure you have configured email address as a username option. Learn more about [multi-attribute login]({{base_path}}/guides/authentication/multi-attribute-login/).
```

Comment:
should use the doc: https://is.docs.wso2.com/en/latest/guides/users/attributes/enable-email-as-username/ similar to above

Reply:
fixed with 3033eb4d1d38180c4ca1d0ad42b202f75b46363f